From 3811859616f5f36dad13107602dca721f271e33c Mon Sep 17 00:00:00 2001
From: Tyler Nijmeh <tylernij@gmail.com>
Date: Sat, 18 May 2019 14:38:17 -0700
Subject: [PATCH] fs: Reduce cache pressure

We can more utilize our available RAM better like this.

Signed-off-by: Tyler Nijmeh <tylernij@gmail.com>
---
 fs/dcache.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: common/fs/dcache.c
===================================================================
--- common/fs/dcache.c	2025-10-14 12:24:36.382535951 +1030
+++ common/fs/dcache.c	2025-10-14 12:24:36.374653998 +1030
@@ -71,7 +71,7 @@
  * If no ancestor relationship:
  * arbitrary, since it's serialized on rename_lock
  */
-int sysctl_vfs_cache_pressure __read_mostly = 100;
+int sysctl_vfs_cache_pressure __read_mostly = 50;
 EXPORT_SYMBOL_GPL(sysctl_vfs_cache_pressure);
 
 __cacheline_aligned_in_smp DEFINE_SEQLOCK(rename_lock);
